pom.xml

```diff
@@ -74,6 +74,10 @@
 			<artifactId>liquibase-core</artifactId>
 			<version>${liquibase.version}</version>
 		</dependency>
+		<dependency>
+			<groupId>org.hibernate.validator</groupId>
+			<artifactId>hibernate-validator</artifactId>
```

Comment:
It's preferable to use a starter (`spring-boot-starter-validation`) over specific implementation.